tensorflow_model_optimization/g3doc/guide/install.md

```diff
@@ -2,6 +2,11 @@
 
 ### Stable Builds
 
+It is recommended to create a Python virtual environment before proceeding to
```

Comment:
Could you move this above the ### Stable Builds header? 

This comment is applicable to both Stable Builds and Installing from Source? 

Afterwards, squash the commits and we can merge the PR. Thanks for the PR.

Reply:
Effectively it's applicable to both. I made the modification, thanks.